app/controllers/guest_rooms_controller.rb

```diff
@@ -0,0 +1,14 @@
+class GuestRoomsController < ApplicationController
```

Comment:
Excellent work setting up a joins controller for the creation of the joins record! This is the most RESTful design option here IMO